source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,251 @@
+=======================
+Halloween Treasure Hunt
```

Comment:
For the sake of organization and to make the figure naming less confusing, let's also number the example projects. It can't hurt.

So in this case, let's make this section `1. Halloween Treasure Hunt`.